flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/adaptive/CreatingExecutionGraphTest.java

```diff
@@ -166,18 +170,19 @@ static class MockCreatingExecutionGraphContext extends MockStateWithoutExecution
                 new StateValidator<>("Executing");
 
         private Function<
-                        CreatingExecutionGraph.ExecutionGraphWithVertexParallelism,
-                        CreatingExecutionGraph.AssignmentResult>
+                CreatingExecutionGraph.ExecutionGraphWithVertexParallelism,
+                CreatingExecutionGraph.AssignmentResult>
                 tryToAssignSlotsFunction =
-                        e -> CreatingExecutionGraph.AssignmentResult.success(e.getExecutionGraph());
+                e -> CreatingExecutionGraph.AssignmentResult.success(e.getExecutionGraph());
 
         private GlobalFailureHandler globalFailureHandler =
                 t -> {
                     // No-op.
                 };
 
         public void setExpectWaitingForResources() {
-            waitingForResourcesStateValidator.expectInput((none) -> {});
+            waitingForResourcesStateValidator.expectInput((none) -> {
```

Comment:
uuh, what's happening here 🤔 did we update formatter?

Reply:
No i screwed up :) My intellij is using a spotless config that isnt fully compatible with Flink...